velox/functions/prestosql/Repeat.cpp

```diff
@@ -22,6 +22,10 @@ namespace {
 // See documentation at https://prestodb.io/docs/current/functions/array.html
 class RepeatFunction : public exec::VectorFunction {
  public:
+  RepeatFunction() = default;
```

Comment:
By convention, we place shared functions under velox/functions/lib.

See velox/functions/lib/Re2Functions.h

We don't need default constructor.